site/src/docs/components/highlight/tabs.mdx

```diff
@@ -0,0 +1,30 @@
+---
+slug: '/components/highlight/tabs'
+title: 'Highlight'
+---
+
+import { StatusLabel } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# Highlight
+
+<StatusLabel type="alert" variant="rounded">Draft</StatusLabel>
+
+<LeadParagraph>
+    Quote and highlight allow lifting an excerpt from a longer text to emphasize. The Quote element has a reference to the quote source. The Highlight element is an excerpt from the text without the source information.
```

Comment:
Maybe mention here straight away that this Highlight component contains quote styles as well. Something like: `Highlight component allows lifting an excerpt from a longer text for emphasis. Highlight supports two types: highlight (default) and quotation.--` and then the two sentences you have here?